src/EventStore.Core.Tests/ClientOperations/when_commiting_a_transaction_with_data.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
```

Comment:
Fix spelling of `committing` in the filename and class.

Reply:
fixed